### ISSUE TITLE
EN - Go through the product to replace "click" with a more inclusive term

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#2003]
We use "click" in some product communications. "Select" or "Use are more inclusive terms.
(Creating card for this as directed by Carolyn )

<img width="930" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeTF1Q2c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--fc57499322a231861cf1f1aa1212629c9555355b/Screenshot%202025-06-18%20at%201.14.24%E2%80%AFPM.png" alt="Screenshot 2025-06-18 at 1.14.24 PM.png" />

## Potential fix

- What change could fix this issue?
- What nuggets of information could help us fix the issue

## Acceptance Criteria

- [ ] Should this issue be added to the A11y Statement if it cannot be resolved soon?

**Comment by YedidaZalik** [comment]
List of instances in both languages in Templates and Code
[Audit for “click/cliquer”](https://docs.google.com/document/d/1XbaJNNrE2deA8WmEE08aeMOe-dq5m8wdYsDEuwTl9Ms/edit?tab=t.0#heading=h.dscc54dp9uzt)

**Comment by YedidaZalik** [comment]
Decided on replacements. In many case, we already had the FR in the code. There's a few I've highlighted to check with Phil.

**Comment by yaelberger-commits** [comment]
@amazingphilippe what would be the next steps on this task?

**Comment by YedidaZalik** [comment]
updated template content

**Comment by amazingphilippe** [comment]
PR has been merged 2 weeks ago: https://github.com/cds-snc/notification-admin/pull/2394
